### ISSUE TITLE
PostgresDatabase should ignroe addPrimeryKey clustered false

### ISSUE THREAD

**Issue by uwe42** [liquibase/liquibase#1779]
## Environment
**Liquibase Integration & Version**: spring boot 2.4.2

**Database Vendor & Version**: postgresql

**Operating System Type & Version**: all

## Description
The [documentation ](https://docs.liquibase.com/change-types/community/add-primary-key.html) states that clustered is supported by all dbms. 

Not all databases support clustered primary keys, but Postgresql is one that does. The code to set the primary key as clustered exists in addPrimaryKey, but the validation check blocks it from being run. The validation also fails if clustered is set to "false".

## Steps To Reproduce
```yaml
- changeSet:
    id: 1-pkey
    author: me
    changes:
    - addPrimaryKey:
        tableName: testTable
        columnNames: id
        constraintName: testTable_pkey
        clustered: false
```

## Actual Behavior
```
liquibase.exception.ValidationFailedException: Validation Failed:
     1 changes have validation failures
          clustered is not allowed on postgresql [...]
```

## Expected/Desired Behavior
* Running 'liquibase update' should run without failure and create the primary key as clustered or not depending on the clustered argument

**Comment by molivasdat** [comment]
Hi @uwe42 Thanks for alerting us to this issue.  We will update the documentation to add which databases are supported for clustering.
Here is a link to our documentation.  You can close this one and you can add an issue there or fix it with a PR directly.
[https://github.com/Datical/liquibase-docs](https://github.com/Datical/liquibase-docs)

**Comment by uwe42** [comment]
It could be ignored for postgresql, that would simplify the create of changesets for multiple databases.

**Comment by nvoxland** [comment]
I found that clustered should be supported with postgresql but the validation was blocking it. 

I fixed the validation logic in #1880
